ld/src/ld/parsers/macho_relocatable_file.cpp

```diff
@@ -3119,6 +3126,13 @@ const char* Parser<A>::scanSymbolTableForAddress(uint64_t addr)
 {
 	uint64_t closestSymAddr = 0;
 	const char* closestSymName = NULL;
+
+	const macho_nlist<P>& sym =	_symbolsMap[addr];
```

Comment:
if addr isn't present, this will insert a new element (with the default constructor iirc). it looks like there are cases where addr isn't present, so i think we want .find